lib/simple_form/inputs/color_input.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module SimpleForm
+  module Inputs
+    class ColorInput < Base
+      def input(wrapper_options = nil)
+        input_html_options[:type] ||= "color" if html5?
```

Comment:
Maybe we don't need to check the `html5?` and always generate the input with the `type="color"`. The responsibility of using it or not would stay with the developer.

Reply:
I was following the same patterns as [range](https://github.com/plataformatec/simple_form/blob/master/lib/simple_form/inputs/range_input.rb) or [numeric](https://github.com/plataformatec/simple_form/blob/master/lib/simple_form/inputs/numeric_input.rb) but if we don't mind for color then I can change it

Reply:
So let's keep as it is :)